pkg/apimachinery/registered/registered.go

```diff
@@ -99,12 +121,6 @@ func IsEnabledVersion(v unversioned.GroupVersion) bool {
 	return found
 }
 
-// IsRegisteredVersion returns if a version is registered.
-func IsRegisteredVersion(v unversioned.GroupVersion) bool {
```

Comment:
Yes right now availabeVersions, enabledVersions and groupMeta have overlapping information.
I plan to refactor that in another PR (this PR is just code moving, no logic change).
We will see what methods are required after the refactor and add any thats missing.

No point in keeping dead code around.